site/src/sections/hero.style.js

```diff
@@ -0,0 +1,129 @@
+import styled from "styled-components";
+
+const HeroStyleWrapper = styled.div`
+    height: 53rem;
+    width: 100%;
+    background-color: #00B39F;
+    color: #FFF;
+    font-family: 'Qanelas Soft', 'Open Sans', sans-serif;
```

Comment:
No need to define `font-family` again anywhere in the code. It is controlled centrally in app.style.js

Reply:
Resolved